M2/Macaulay2/packages/NormalToricVarieties/ToricVarieties.m2

```diff
@@ -693,13 +693,13 @@ toricBlowup (List, NormalToricVariety, List) := NormalToricVariety => (s, X, v)
       	    if member (s#0,t) then continue
       	    else sort (t | s)
 	    );
-    	return normalToricVariety (rays X, coneList | coneList') 
+	return normalToricVariety (rays X, coneList | coneList', CoefficientRing => X.cache.CoefficientRing, Variable => X.cache.Variable)
 	);
     coneList' = for t in clStar list (
 	if all (s, i -> member (i,t)) then continue
 	else t | {n}
 	);
-    Z := normalToricVariety (rays X | {v}, coneList | coneList');
+    Z := normalToricVariety (rays X | {v}, coneList | coneList', CoefficientRing => X.cache.CoefficientRing, Variable => X.cache.Variable);
     Z.cache.toricBlowup = X;
```

Comment:
Do you know why the base `X` is cached here but not above?

Reply:
Not sure I understand the question - where above would/should X be cached?

Reply:
Oh, I see now - I'm not sure why that differs. I could add the base into the cache, unless @ggsmith or anyone else has objections?

Reply:
For consistency, I think that the base toric variety should also be cached in the ```#s === 1``` case.  @Devlin-Mallory can you also make this change?

Reply:
Done!